### DISCUSSION TITLE
V2.8.0

### DISCUSSION THREAD

**Discussion by Joe91** [Joe91/fun-bots#316] [Announcements]
[![Support Server](https://img.shields.io/discord/862736286774198322.svg?label=Discord&logo=Discord&colorB=7289da&style=for-the-badge)](https://discord.com/invite/FKamccAEqz)
[![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg?style=for-the-badge)](https://www.paypal.me/joe91de)
![Image](https://img.shields.io/github/downloads/Joe91/fun-bots/total?style=for-the-badge)
![Image](https://img.shields.io/github/stars/Joe91/fun-bots?style=for-the-badge)

## Welcome to the changelogs for release **V2.8**
This is the changelog for the version V2.8 Don't forget to [join us on Discord](https://discord.com/invite/FKamccAEqz)

### New features / improvements
* remove Zombie-mode (will soon be a separated mod, currently on a branch)
* real Hitboxes thanks to crazy workaround by Bree_Arnold
* option to use old hitboxes (and enable bots to use the knive again) in Registry added
* fix and update permissions in UI
* add option for vehicles to move while attacking (default active)
* improve rocket- and nade-handling
* fix for end-of-round-crash on conquest
* Option in Registry added to prevent bots from spawning on the last ticket in conquest
* improve compatibility to more-gore-mode
* vehicle-weapon-improvement and code-cleanup by tagener-noisu #295
* Real weapondata and aimin in Gunmaster #292
* Merged pull request #301 from tagener-noisu min nade distance for bots (also with Registry-Entry)
* Merged pull request #300 from tagener-noisu trace navigation
* Mergeed pull request #303 from tagener-noisu ifv_tows weapon usage
* Offset of vehicle-weapons was added
* Kill inactive bots after some time (#307)
* Merged pull request #314 from tagener-noisu Change target waypoint distance for air vehicles 

### Bug fixes
* fix meele attack
* missing nil-check in vehicle-movement
* fix bots getting stuck on vehicle-paths #297
* overwrite of loop / reverse path does not work #304
* fix GM-Rocket-usage #305

### New maps
* Gunmaster for MP_001 by Augusta
* Gunmaster for MP_003 by Augusta
* Gunmaster for MP_007 by Augusta
* Gunmaster for MP_011 by Augusta
* Gunmaster for MP_012 by Augusta
* Gunmaster for MP_018 by Augusta
* GunMaster for XP1_003 by Volbeat13
* Rush for XP4_FD by Rolling Wave
* Rush for XP4_Parl by Rolling Wave
* Rush for XP4_Quake by Rolling Wave
* Rush for XP3_Desert by Rolling Wave
* Rush for XP3_Valley by Rolling Wave
* SquadRush for MP_011 by Odysseus
* SquadRush for XP3_Shield by Odysseus
* Scavenger for XP4_Rubble by Odysseus
* SquadRush for XP4_Quake by Volbeat13
* SquadRush for XP3_Valley by day of light
* Rush for XP5_001 by Volbeat13
* Rush for XP5_002 by Sunseeker
* Rush for XP5_003 by Sunseeker
* Rush for XP3_Shield by Sunseeker
* SquadRush XP2_Factory by Sunseeker
* SquadRush XP2_Office by Sunseeker
* SquadRush XP5_002 by Sunseeker
* SquadRush XP5_004 by Sunseeker
* SquadRush for XP3_Alborz by Every_knows9
* SquadRush for MP_001 by Every_knows9
* SquadRush for MP_003 by Every_knows9
* SquadRush for MP_007 by Every_knows9
* SquadRush for MP_017 by Every_knows9
* SquadRush for MP_Subway by Every_knows9
* SquadRush for XP3_Desert by Every_knows9

### Updated maps
* Rush for MP_001 by Rolling Wave
* Rush for MP_012 by Rolling Wave
* Rush for MP_013 by Rolling Wave
* Rush for MP_003 by Volbeat13
* Rush for MP_011 by Odysseus
* Rush for MP_017 by Odysseus
* Rush for MP_018 by Odysseus
* TDM XP2_Factory Rework by Sunseeker
* Rush for XP1_003 Rework by Sunseeker
* Rush for XP1_001 Rework by Sunseeker
* Rush for XP4_Quake Rework by Sunseeker
* Conquest Large for MP_011 by ThyKingdomCome

### Documentation update

<hr /><em>This discussion was created from the release <a href='https://github.com/Joe91/fun-bots/releases/tag/V2.8.0'>V2.8.0</a>.</em>

**Comment by yaser5123** [comment]
Thank you for this amazing mod
